icons/docusign.svg

```diff
@@ -1 +1 @@
-<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>DocuSign</title><path d="M9.517 3.31h4.966v6.621h3.31L12 16.552 6.207 9.931h3.31V3.31zM0 19.034h24v1.655H0v-1.655z"/></svg>
\ No newline at end of file
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>DocuSign</title><path d="M7.298.006a.582.582 0 0 0-.583.583v6.12h6.493a.692.692 0 0 1 .496.21l3.37 3.371c.136.124.21.31.21.496v6.492C20.99 17.266 24 13.784 24 8.63c0-5.167-3.308-8.636-7.62-8.624Zm9.987 17.272h-10a.582.582 0 0 1-.582-.582V6.722H.583A.582.582 0 0 0 0 7.304v16.108c0 .322.26.582.582.582h16.12c.322 0 .583-.26.583-.582z"/></svg>
```

Comment:
You'll need to update the capitalization here as well, sorry.

Reply:
oh yes, i forgot to update this